### DISCUSSION TITLE
Signing keys from Cloudsmith not available anymore

### DISCUSSION THREAD

**Discussion by ghimisradu** [rabbitmq/rabbitmq-website#1716] [General]
### Describe the bug

Following the installation guide from [here](https://www.rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith), the gpg.E495BB49CC4BBE5B.key and gpg.9F4587F226208342.key are nota available anymore(404 status).

### Reproduction steps

1) go to https://www.rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith
2) follow the install script steps
3) get stuck at " Community mirror of Cloudsmith: modern Erlang repository"

### Expected behavior

The signing keys getting installed successfully

### Additional context

Not sure if I should posted here or on rabbitmq-server repo.

**Comment by michaelklishin** [comment]
`https://ppa.novemberain.com/gpg.E495BB49CC4BBE5B.key` downloads just fine for me.

**Comment by michaelklishin** [comment]
And does does

```
curl -sL 'https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/gpg.E495BB49CC4BBE5B.key'
```

(the same key served directly by Cloudsmith).

The issue is on your end.

That said, I see that the key was never added to https://github.com/rabbitmq/signing-keys, which I will do soon.

**Reply by michaelklishin** [reply]
Voilà: [all public signing keys used by Team RabbitMQ](https://github.com/rabbitmq/signing-keys/releases/tag/3.0) in one place.

If you have reasons to do so, you can update the installation instructions to download the key from GitHub.
